lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -422,11 +422,13 @@ def draw(self, renderer):
             pos[index] = tick.get_loc()
             pos[tickdir] = (
                 edgep1[tickdir]
-                + info['tick']['outward_factor'] * ticksign * tickdelta)
+                + info['tick']['outward_factor'] * ticksign * tickdelta
+                * (0 if tick._tickdir == 'out' else 1))
```

Comment:
I think something like this would be clearer?
```suggestion
            pos[tickdir] = edgep1[tickdir]
            if tick._tickdir != 'out':
                pos[tickdir] += info['tick']['outward_factor'] * ticksign * tickdelta
```

Reply:
sure let me test it out and update.

Reply:
updated and tested - it works. thanks